doc/ref_cert/RC2/chapters/chapter04.md

```diff
@@ -20,7 +20,7 @@ The following is a Requirements Traceability Matrix (RTM) mapping Test Case, and
 
 ### Test Case Traceability to RA2 Requirements
 
-This section focuses on the test cases covering the requirements in [RA-2 Chapter 4.9](../../../ref_arch/kubernetes/chapters/chapter04.md#49-kubernetes-workloads) for Kubernetes workloads.
+This section focuses on the test cases covering the requirements in [RA-2 Chapter 4.9](../../../ref_arch/kubernetes/chapters/chapter04.md) for Kubernetes workloads.
```

Comment:
The problem is that now the reference is to the main chapter and someone has to search to go to a specific section. Until RA2 is converted, the reference should remain. To make it easy to change, maybe the references can be named and listed sperartely, and we will only need to change in one easily visible place.

Reply:
I agree with Pankaj here

Reply:
As far as I understand I cannot be more precise with markdown. Let me double check why linkcheck is considering this link as false.
Yes it's simple in RST.